src/libraries/Common/src/System/Drawing/KnownColor.cs

```diff
@@ -200,6 +200,7 @@ enum KnownColor
         GradientActiveCaption,
         GradientInactiveCaption,
         MenuBar,
-        MenuHighlight
+        MenuHighlight,
```

Comment:
We can delete the comment above about "New system color additions...".  That's ancient history, and is no longer accurate with these additions.

Reply:
i removed the comment